management.md

```diff
@@ -35,5 +35,48 @@
 - What process will the team follow to find, report, and fix defects in the
   software project?
 
-- How will your team know when it is finished with the implementation of the
-  software project?
+  - Finding Defects:
+
+    In order to find the defects in a software project,
+    first check the error
+    messages terminal. Secondly, if there still are
+    effects, check the test
+    cases and asses their correctness. Lastly,
+    check TravisCI to see which
```

Comment:
There should be a space between "Travis" and "CI". Also, it is worth discussing if you will use `Travis CI` or Travis CI in the future